CtbGui

```diff
@@ -1971,7 +1989,13 @@ class MainWindow(QtWidgets.QMainWindow):
                     if self.comboBoxPlot.currentIndex() == Defines.ImageIndex.Matterhorn.value:
                         self.matterhornImage(trans_array)
 
+                    # keep the zoomed in state
+                    if not self.firstImage:
+                        viewBox.setState(state)
+
 
+            if self.firstImage:
```

Comment:
I think firstImage should only be set to False if the analog plot is enabled. First doing an acq without plotting now changes how the zoom looks when you switch on plotting.

Reply:
done. each readoutmode have different image widgets and so each of them have a first image variable.